documentation/modules/proc-deploying-api-server.adoc

```diff
@@ -3,15 +3,16 @@
 // assembly-installing-manual-steps.adoc
 
 [id='deploying-api-server-{context}']
-= (Optional) Deploying the API server
+= Deploying the API server
 
-The API server provides a REST API for creating address spaces and addresses. It can also serve as
-a Kubernetes API server if it is registered as an APIService.
+The API server provides a REST API for creating address spaces and addresses. It can also serve as an aggregated API server if it is registered as an APIService.
 
-NOTE: To install {ProductName} on {PlatformName}, you must have `cluster-admin` access to set up the
-required roles for delegating authentication to the Kubernetes master; otherwise, you are restricted
-to a single address space. For more information about how to deploy without `cluster-admin` access,
-which restricts {ProductName} to a single address space, see xref:deploying-single-namespace-{context}[].
+ifeval::["{PlatformName} == "OpenShift"]
+NOTE: To install {ProductName} API Server on {PlatformName}, you must have `cluster-admin` access to set up the
```

Comment:
Change NOTE: to .Prerequisites
(Current template guidance is to use the plural, even if just one prereq is specified

To install the {ProductName}